test/job.js

```diff
@@ -15,7 +15,7 @@ describe('Job.js Tests:', function(){
     });
 
 
-    describe('Job Constructor Tests:', async function(){
+    describe('Job Constructor Tests:', function(){
```

Comment:
All the changes on this file are not related to this issue. 

Please revert the file back to its original state in this branch.

Reply:
Reverted. Done.